src/panel/pages/timeline/Timeline.fixture.tsx

```diff
@@ -7,7 +7,7 @@ const defaultEvents: ReceivedDebugEvent[] = [
   {
     type: "debug",
     data: {
-      type: "addition",
+      type: "query",
```

Comment:
I'm not totally sure whether these will change - I think we should work it out on a call some time today!